apps/src/aiTutor/interactionsApi.ts

```diff
@@ -1,23 +1,23 @@
-import MetricsReporter from '../lib/metrics/MetricsReporter';
 import {getAuthenticityToken} from '../util/AuthenticityTokenStore';
 import {AITutorInteraction} from './types';
+import MetricsReporter from '@cdo/apps/lib/metrics/MetricsReporter';
 
-export async function savePromptAndResponse(interactionData: AITutorInteraction) {
+export async function savePromptAndResponse(
+  interactionData: AITutorInteraction
+) {
   try {
-    let response = await fetch('/ai_tutor_interactions', {
+    await fetch('/ai_tutor_interactions', {
       method: 'POST',
       headers: {
         'Content-Type': 'application/json',
         'X-CSRF-Token': await getAuthenticityToken(),
       },
       body: JSON.stringify(interactionData),
     });
-    let data = await response.json();
-    console.log('data', data);
   } catch (error) {
     MetricsReporter.logError({
-      event: 'AI Tutor interaction failed to save.',
-      error: error,
+      event: 'AI Tutor Interaction failed to save.',
```

Comment:
We've started putting constants for events [here](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/lib/metrics/events.ts), I'd also make this more easily searchable by changing the event name to something like `AI_TUTOR_SAVE_FAIL`.

Reply:
Noted. I have few 🧹 clean up items like this and `RootState` that I'll pull into a follow up  PR to get https://github.com/code-dot-org/code-dot-org/pull/56361 unblocked. Thanks!